### PR TITLE
[Bug 1047902] Add volunteer button to the l10n dashboard

### PR DESCRIPTION
**Pull request by MikkCZ**
Needs some tests.

### CODE REVIEW

**Comment by MikkCZ** [comment]
And maybe I can make a macro for that button common with the landings page, because the grid class does not seem to have any effect there.

**Comment by mythmon** [comment]
@MikkCZ I don't think you need any tests for this. There isn't any logic; it is just a button.

I'm going to delegate review access to you for this PR. If you want to merge this as is, say "r+" to the [homu bot](https://homu.io) bot, and it will be merged. If you want to add tests, I can review those too. (You can remove your delegation power by saying `delegate-`)

@homu delegate+

**Comment by MikkCZ** [comment]
Thank you. I would like to create a test that the button is shown for not logged users only.

**Comment by MikkCZ** [comment]
I was not sure if to put the tests in `test_views.py` or `test_templates.py`, but `test_views.py` wins at the end as `LocalizationDashAnnouncementsTests` seems more similar (despite I have a copied the test approach from `kitsune/wiki/tests/test_templates.py:NewRevisionTests.test_draft_button`).

BTW I noticed line 26 in `test_views.py`. Shouldn't be `class LocalizationDashAnnouncementsTests(TestCase):` instead of `def LocalizationDashAnnouncementsTests(TestCase):`?

**Comment by safwanrahman** [comment]
As you have made changes in templates, you should add test in ``test_templates.py`` :100:

**Comment by MikkCZ** [comment]
Tests moved.

File: kitsune/dashboards/tests/test_templates.py [hunk 1]
```diff
@@ -35,3 +36,23 @@ def test_render(self):
         eq_(200, response.status_code)
         doc = pq(response.content)
         self._assert_readout_contains(doc, 'unreviewed', unreviewed.document.title)
+
+
+class LocalizationDashVolunteerButtonTests(TestCase):
```
Comment (safwanrahman) [inline kitsune/dashboards/tests/test_templates.py]: I think, you can insert it in ``LocalizationDashTests`` class. having another class seems to be overkill. as its only testing the Dashboard thing

Comment (MikkCZ) [inline kitsune/dashboards/tests/test_templates.py]: Done

**Comment by safwanrahman** [comment]
Thats Awesome. r+ from my end.

**Comment by mythmon** [comment]
@homu r=safwanrahman
@homu delegate=safwanrahman

**Comment by safwanrahman** [comment]
@homu retry

**Comment by safwanrahman** [comment]
@homu retry

**Comment by mythmon** [comment]
@homu retry

**Comment by mythmon** [comment]
That happened because @Osmose merged something manually instead of with homu. Restarting this will make it pick up the new merged one and work this time.

@homu retry

**Pull request merged**